.github/workflows/packaging.yml

```diff
@@ -11,7 +20,8 @@ jobs:
       github.event.pull_request.head.repo.full_name != github.repository
     # Packaging for CentOS 7 does not work with other versions, see:
     # https://github.com/packpack/packpack/issues/145
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
```

Comment:
Please, move all maintenance version updates to a separate commit.

Reply:
Done, rebased to `master` that has this change